integration-requirements.txt

```diff
@@ -2,7 +2,7 @@
 behave
 jsonschema
 PyHamcrest
-pycloudlib @ git+https://github.com/canonical/pycloudlib.git@44206bb95c49901d994c9eb772eba07f2a1b6661
```

Comment:
Are we updating the pycloudlib version here because we need a feature or just because it doesn't break anything by doing it ?

Reply:
Ah I don't think we actually need it, I thought it might help with an issue but it was unnecessary. I'll revert it back